tests/common/platform/device_utils_dpu.py

```diff
@@ -0,0 +1,123 @@
+"""
+Helper script for DPU  operations
+"""
+
+import time
+import pytest
+from tests.common.utilities import wait_until
+from tests.common.helpers.platform_api import chassis, module
+from tests.platform_tests.api.conftest import platform_api_conn
+from tests.common.devices.sonic import * #noqa
+from tests.common.helpers.assertions import pytest_assert
+
+@pytest.fixture(scope='function', autouse=True)
+def dpu_poweron_fixture(duthosts, enum_rand_one_per_hwsku_hostname, request, platform_api_conn):
```

Comment:
Usually we don't have the "fixture" in a fixture name.

Reply:
Changed it.